extensions/amp-carousel/0.1/scrollable-carousel.js

```diff
@@ -267,8 +267,20 @@ export class AmpScrollableCarousel extends BaseCarousel {
   cancelTouchEvents_() {
     // TODO(aghassemi, #4754): Ideally we only stop propagation of horizontal
     // touchmove events.
-    this.element.addEventListener('touchmove', event => {
-      event.stopPropagation();
-    });
+
+    // We don't preventDefault, tell the browser to optimize scrolling.
+    const options = {
+      passive: true,
+    };
+
+    /**
+     * TODO(aghassemi, #5105): Type check
+     * @suppress {checkTypes}
```

Comment:
What's complication with type checking? Is it still not in externs? If so, please file a bug against closure.